### PR TITLE
[docs][PLAT-7408] Add details of certified Helm charts for OpenShift

### PR DESCRIPTION
**Pull request by bhavin192**
This change adds instructions related to Red Hat certified Helm charts for OpenShift environment.
- Removes the operator based installation from preview as we never had preview releases of operator available.
- Deprecate the operator based installation on stable, it is not maintained and we don't have any users.

Have tested the instructions by following the steps on an OpenShift cluster.

https://deploy-preview-19701--infallible-bardeen-164bc9.netlify.app/stable/yugabyte-platform/install-yugabyte-platform/install-software/openshift/
https://deploy-preview-19701--infallible-bardeen-164bc9.netlify.app/preview/yugabyte-platform/install-yugabyte-platform/install-software/openshift/

### CODE REVIEW

File: docs/content/stable/yugabyte-platform/install-yugabyte-platform/install-software/openshift.md [hunk 1]
```diff
@@ -51,264 +51,124 @@ As YugabyteDB Anywhere was formerly called Yugabyte Platform, you might see the
...
+
+## Helm-based installation
 
 Installing YugabyteDB Anywhere on an OpenShift cluster using Helm involves the following:
 
-- [Creating an instance of YugabyteDB Anywhere](#create-an-instance-of-yugabyte-platform-via-helm)
+- [Creating an instance of YugabyteDB Anywhere](#create-an-instance-of-yugabytedb-anywhere-via-helm)
 - [Finding the availability zone labels](#find-the-availability-zone-labels)
-- [Configuring the CLI with the OCP cluster](#configure-the-cli-with-the-ocp-cluster)
 - [Accessing and configuring YugabyteDB Anywhere](#access-and-configure-yugabyte-platform)
```
Comment (ddhodge) [inline docs/content/stable/yugabyte-platform/install-yugabyte-platform/install-software/openshift.md:166]:
```suggestion
- [Accessing and configuring YugabyteDB Anywhere](#access-and-configure-yugabytedb-anywhere)
```

File: docs/content/stable/yugabyte-platform/install-yugabyte-platform/install-software/openshift.md [hunk 2]
```diff
@@ -380,6 +240,8 @@ To create a YugabyteDB Anywhere instance, perform the following:
   - ClusterRole or ClusterRoleBinding with the cluster-admin role are recommended if your intention is to create clusters across multiple namespaces.
   - Certain container-level metrics like CPU, memory will be unavailable without the above `ClusterRoleBinding`.
```
Comment (ddhodge) [inline docs/content/stable/yugabyte-platform/install-yugabyte-platform/install-software/openshift.md:241]:
```suggestion
If you don't create the `ClusterRoleBinding`, some container-level metrics like CPU and memory will be unavailable.
```

File: docs/content/stable/yugabyte-platform/install-yugabyte-platform/install-software/openshift.md [hunk 3]
```diff
@@ -380,6 +240,8 @@ To create a YugabyteDB Anywhere instance, perform the following:
   - ClusterRole or ClusterRoleBinding with the cluster-admin role are recommended if your intention is to create clusters across multiple namespaces.
```
Comment (ddhodge) [inline docs/content/stable/yugabyte-platform/install-yugabyte-platform/install-software/openshift.md:240]:
```suggestion
Creating ClusterRole or ClusterRoleBinding with the cluster-admin role are recommended if your intention is to create clusters across multiple namespaces.

```

File: docs/content/stable/yugabyte-platform/install-yugabyte-platform/install-software/openshift.md [hunk 4]
```diff
@@ -484,3 +342,209 @@ Expect the following output:
 ```output
 12.34.56.78
 ```
+
+## (Deprecated) Operator-based installation {#operator-based-installation}
+
+{{< warning title="Operator based installation is deprecated" >}}
+The operator based installation of YugabyteDB Anywhere has been deprecated, consider using the [Red Hat certified Helm charts](#certified-helm-chart-based-installation) instead.
```
Comment (ddhodge) [inline docs/content/stable/yugabyte-platform/install-yugabyte-platform/install-software/openshift.md:349]:
```suggestion
Operator-based installation of YugabyteDB Anywhere has been deprecated, consider using the [Red Hat certified Helm charts](#certified-helm-chart-based-installation) instead.
```

File: docs/content/stable/yugabyte-platform/install-yugabyte-platform/install-software/openshift.md [hunk 5]
```diff
@@ -484,3 +342,209 @@ Expect the following output:
...
+{{< /warning >}}
+
+Installing YugabyteDB Anywhere on an OpenShift cluster using the YugabyteDB Anywhere Operator involves the following:
+
+- [Installing the Operator itself](#install-the-operator)
```
Comment (ddhodge) [inline docs/content/stable/yugabyte-platform/install-yugabyte-platform/install-software/openshift.md:354]:
```suggestion
- [Installing the Operator](#install-the-operator)
```

File: docs/content/stable/yugabyte-platform/install-yugabyte-platform/install-software/openshift.md (hunk 3 above)
Comment (bhavin192) [reply docs/content/stable/yugabyte-platform/install-yugabyte-platform/install-software/openshift.md:240]: Actually, this sentence shouldn't be here. I will remove it.

File: docs/content/preview/yugabyte-platform/install-yugabyte-platform/install-software/kubernetes.md [hunk 6]
```diff
@@ -394,6 +394,7 @@ The PostgreSQL and Nginx containers always run as non-root. To run the rest of t
 securityContext:
   enabled: true
 ```
+This value is not supported on OpenShift as it runs all the containers of YugabyteDB Anywhere as non-root by default. Modifying this on OpenShift could cause the containers to fail.
```
[the remark this answers is not in the document]
Comment (bhavin192) [reply docs/content/preview/yugabyte-platform/install-yugabyte-platform/install-software/kubernetes.md]: Makes sense, thanks!

File: docs/content/preview/yugabyte-platform/install-yugabyte-platform/install-software/openshift.md [hunk 7]
```diff
@@ -52,264 +52,68 @@ As YugabyteDB Anywhere was formerly called Yugabyte Platform, you might see the
...
-```
+## Certified Helm chart based installation
 
-To verify that the pods are being updated, execute the following command:
+{{< note title="Note" >}}
 
-```shell
-oc get pods -n yb-platform -l app=ybplatform-sample-yugaware -w
-```
+The YugabyteDB Anywhere [Red Hat certified Helm chart](https://catalog.redhat.com/software/container-stacks/detail/6371ef8950ac9be93651a765) is available for the stable versions. See [Certified Helm chart based installation](/stable/yugabyte-platform/install-yugabyte-platform/install-software/openshift/#certified-helm-chart-based-installation) in the stable version's documentation for the instructions.
```
[the remark this answers is not in the document]
Comment (bhavin192) [reply docs/content/preview/yugabyte-platform/install-yugabyte-platform/install-software/openshift.md:100]: I see, didn't realize the fact that we copy preview to stable when a new version is released. 
>  Is it completely not possible to install a preview version using a helm chart?

We are not certifying the preview versions through Red Hat, we might do it later once we have automation in place. Also, there is some bug in OpenShift which shows only the latest version in the UI, so if we release preview version, users won't be able to install the stable versions. 

How about keeping the instructions in both preview and stable with a small note that only stable versions can be installed for the certified Helm charts?

[the remark this answers is not in the document]
Comment (ddhodge) [reply docs/content/preview/yugabyte-platform/install-yugabyte-platform/install-software/openshift.md:100]: That would work

**Review by iSignal — APPROVED** [review]
Looks good!

File: docs/content/preview/yugabyte-platform/install-yugabyte-platform/install-software/openshift.md [hunk 8]
```diff
@@ -381,6 +252,8 @@ To create a YugabyteDB Anywhere instance, perform the following:
   - ClusterRole or ClusterRoleBinding with the cluster-admin role are recommended if your intention is to create clusters across multiple namespaces.
   - Certain container-level metrics like CPU, memory will be unavailable without the above `ClusterRoleBinding`.
```
Comment (bhavin192) [inline docs/content/preview/yugabyte-platform/install-yugabyte-platform/install-software/openshift.md:253]:
```suggestion

If you don't create the `ClusterRoleBinding`, some container-level metrics like CPU and memory will be unavailable.
```

**Comment by bhavin192** [comment]
@ddhodge thank you for the review! I made a small change, can you take a look at it and feel free to merge this if it looks good. Or I will merge it once you approve the PR.

File: docs/content/preview/yugabyte-platform/install-yugabyte-platform/install-software/openshift.md [hunk 9]
```diff
@@ -52,263 +52,134 @@ As YugabyteDB Anywhere was formerly called Yugabyte Platform, you might see the
...
 
- Expect the following output:
+Installing the YugabyteDB Anywhere [Red Hat certified Helm chart](https://catalog.redhat.com/software/container-stacks/detail/6371ef8950ac9be93651a765) on an OpenShift cluster involves the following:
 
-```output
-ybplatform.yugabyte.com/ybplatform-sample patched
-```
+- [Installing YugabyteDB Anywhere using certified Helm chart](#install-yugabytedb-anywhere-using-certified-helm-chart)
+- [Finding the availability zone labels](#find-the-availability-zone-labels)
+- [Accessing and configuring YugabyteDB Anywhere](#access-and-configure-yugabyte-platform)
```
Comment (ddhodge) [inline docs/content/preview/yugabyte-platform/install-yugabyte-platform/install-software/openshift.md:109]:
```suggestion
- [Accessing and configuring YugabyteDB Anywhere](#access-and-configure-yugabytedb-anywhere)
```

File: docs/content/stable/yugabyte-platform/install-yugabyte-platform/install-software/openshift.md [hunk 10]
```diff
@@ -51,263 +51,134 @@ As YugabyteDB Anywhere was formerly called Yugabyte Platform, you might see the
...
 
- Expect the following output:
+Installing the YugabyteDB Anywhere [Red Hat certified Helm chart](https://catalog.redhat.com/software/container-stacks/detail/6371ef8950ac9be93651a765) on an OpenShift cluster involves the following:
 
-```output
-ybplatform.yugabyte.com/ybplatform-sample patched
-```
+- [Installing YugabyteDB Anywhere using certified Helm chart](#install-yugabytedb-anywhere-using-certified-helm-chart)
+- [Finding the availability zone labels](#find-the-availability-zone-labels)
+- [Accessing and configuring YugabyteDB Anywhere](#access-and-configure-yugabyte-platform)
```
Comment (ddhodge) [inline docs/content/stable/yugabyte-platform/install-yugabyte-platform/install-software/openshift.md:108]:
```suggestion
- [Accessing and configuring YugabyteDB Anywhere](#access-and-configure-yugabytedb-anywhere)
```

File: docs/content/stable/yugabyte-platform/install-yugabyte-platform/install-software/openshift.md [hunk 11]
```diff
@@ -406,7 +278,7 @@ You start by logging in the OCP's web console as admin user, and then performing
 
 ### Use the command line
 
-Alternatively, you can find the availability zone codes via the command line.
+Alternatively, you can find the availability zone codes using the oc CLI.
 
 You start by configuring oc with an admin account (kube:admin) and following the procedure described in [Configure oc with the OCP Cluster](#configure-oc-with-the-ocp-cluster).
```
Comment (ddhodge) [inline docs/content/stable/yugabyte-platform/install-yugabyte-platform/install-software/openshift.md:283]: Section linked doesn't exist

File: docs/content/preview/yugabyte-platform/install-yugabyte-platform/install-software/openshift.md [hunk 12]
```diff
@@ -407,7 +279,7 @@ You start by logging in the OCP's web console as admin user, and then performing
 
 ### Use the command line
 
-Alternatively, you can find the availability zone codes via the command line.
+Alternatively, you can find the availability zone codes using the oc CLI.
 
 You start by configuring oc with an admin account (kube:admin) and following the procedure described in [Configure oc with the OCP Cluster](#configure-oc-with-the-ocp-cluster).
```
Comment (ddhodge) [inline docs/content/preview/yugabyte-platform/install-yugabyte-platform/install-software/openshift.md:284]: Section linked doesn't exist

File: docs/content/stable/yugabyte-platform/install-yugabyte-platform/install-software/openshift.md [hunk 13]
```diff
@@ -484,3 +352,209 @@ Expect the following output:
...
+
+  ![Install Operator](/images/ee/openshift-install-operator.png)
+
+After the installation is complete, the message shown in the following illustration is displayed:
+
+![Operator Installed](/images/ee/openshift-operator-installed.png)
+
+#### Use the command line
+
+Alternatively, you can install the operator via the command line. You start by configuring oc with an admin account (kube:admin) and following the procedure described in [Configure oc with the OCP Cluster](#configure-the-cli-with-the-ocp-cluster).
```
Comment (ddhodge) [inline docs/content/stable/yugabyte-platform/install-yugabyte-platform/install-software/openshift.md:394]: Section linked doesn't exist

**Review by ddhodge — COMMENTED** [review]
Some broken links

**Review by ddhodge — APPROVED** [review]

**Pull request merged**